geopandas/tests/test_merge.py

```diff
@@ -48,6 +50,13 @@ def test_concat_axis0(self):
         assert isinstance(res, GeoDataFrame)
         assert isinstance(res.geometry, GeoSeries)
         self._check_metadata(res)
+        exp = GeoDataFrame(pd.concat([pd.DataFrame(self.gdf), pd.DataFrame(self.gdf)]))
+        assert_geodataframe_equal(exp, res)
+        # check metadata comes from first gdf
+        res4 = pd.concat([self.gdf.set_crs("epsg:4326"), self.gdf], axis=0)
+        # Note: this behaviour does not make sense - geom cols combined into one
+        # but they have different CRS and that is lost. We should raise instead.
```

Comment:
Not always. You may know they share the same CRS but one is explicit while the other is not. We should raise only if you have two explicit CRS that are different. Is it listed as an issue somewhere? If not, can you open an issue for it?